frontend/catalyst/compiler.py

```diff
@@ -548,6 +390,43 @@ class Compiler:
     def __init__(self, options: Optional[CompileOptions] = None):
         self.options = options if options is not None else CompileOptions()
 
+    @debug_logger
+    def get_cli_command(self, tmp_infile_name, output_ir_name, module_name, workspace):
+        """Prepare the command for catalyst-cli to compile the file.
+
+        Args:
+            module_name (str): Module name to use for naming
+            workspace (Directory): directory that holds output files and/or debug dumps.
+        Returns:
+            cmd (str): The command to be executed.
+        """
+        cli_build_path = get_bin_path("cli", "DIALECTS_BUILD_DIR") + "/bin/catalyst-cli"
```

Comment:
```suggestion
        cli_build_path = get_bin_path("cli", "DIALECTS_BUILD_DIR") + "/catalyst-cli"
```

Reply:
I would also rename DIALECTS_BUILD_DIR to something else. The dialects build dir specifies that this was done during the build process, but at runtime, the build process does not matter. A better name would be CATALYST_BIN_DIR